napari/components/dims.py

```diff
@@ -219,6 +223,11 @@ def _check_dims(cls, values):
         elif labels_ndim > ndim:
             updated['axis_labels'] = axis_labels[-ndim:]
 
+        # Check the rollable axes tuple has same number of elements as ndim
+        updated['rollable'] = (values['rollable'] + tuple([True] * ndim))[
+            :ndim
+        ]
+
```

Comment:
If we load data with more dimensions, we prefer to have an unlocked first dimension, not shifting locking by new differences in dimensions.  

```suggestion
        updated['rollable'] = (tuple([True] * ndim) + values['rollable'])[
            -ndim:
        ]

```

Reply:
Good catch! Instead of this code change, I suggest using `ensure_len` as I think this general logic is consistent with other attributes handled in this root validator. Looks like we need to fix the `pad_width` argument  name of `ensure_len`, which is actually a default value to pad with (i.e. `True` here).